src/graphql/activityEvents.graphql

```diff
@@ -0,0 +1,54 @@
+query ActivityEvents(
+  $where: activityEventFilter
+  $orderBy: String
+  $orderDirection: String
+) {
+  activityEvents(
+    where: $where
+    orderBy: $orderBy
+    orderDirection: $orderDirection
+    limit: 1000
+    # TODO implement paging
+  ) {
+    items {
```

Comment:
must declare `items` field on queries that return a list

Reply:
got it